Sage/carbon
# update docs for skip link so the menu items are tabbable

### Proposed behaviour



The story for the isSkipLink prop on Link has the skiplink skipping over the menu items - to make the story more realistic, and show the benefit of a skip link, these menu items should be tabbable.

The stories should be written in a `.stories.tsx` file and import into the MDX docs as per recently-agreed practice.

### Current behaviour

The isSkipLink story has the menu items not tabbable, which doesn't really demonstrate the benefits of a skip link for keyboard users.

All stories are written directly in the `.mdx` file and therefore not type-checked.



### Checklist




#### QA


### Additional context



### Testing instructions



There shouldn't be much QA needed here as there are no code changes affecting the component behaviour - only changes to the docs.

Check all the documentation for Link and ensure that the stories are all present, with code samples visible and functioning correctly. In particular check that the menu items on the isSkipLink story are in the keyboard tab order, and are skipped as intended by the skiplink.

The following CodeSandbox is an example of the broken behaviour.
You can see the new behaviour by looking at the version in the comment by `codesandbox[bot]`.
